osu.Game.Modes.Taiko/Objects/Drawables/DrawableHitCircle.cs

```diff
@@ -0,0 +1,196 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using OpenTK;
+using osu.Framework.Graphics.Containers;
+using osu.Game.Modes.Taiko.Objects.Drawables.Pieces;
+using System;
+using System.Collections.Generic;
+using osu.Game.Modes.Objects.Drawables;
+using osu.Framework.Graphics;
+using osu.Framework.Input;
+using OpenTK.Input;
+using OpenTK.Graphics;
+using osu.Game.Graphics;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics.Transforms;
+
+namespace osu.Game.Modes.Taiko.Objects.Drawables
+{
+    public class DrawableHitCircleDon : DrawableHitCircle

```

Comment:
it's okay if:

- it's a nested class
or
- the main class is at the top of the file (right now it's not, leading to confusion
- the main class isn't a large class. in this case it is and would make it hard to find the other two.